### PR TITLE
chore: support for m1 mac

### PR DESCRIPTION
**Pull request by waterfallmouse**
### Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

**Comment by waterfallmouse** [comment]
https://github.com/getsentry/self-hosted/issues/914

**Comment by waterfallmouse** [comment]
This pull request was tentatively created to run this repository on m1 mac.

With this branch, `./install.sh` and `docker-compose up -d` will probably work.

If you have any questions, please feel free to contact me.

**Review by aminvakil — CHANGES_REQUESTED** [review]
AFAICS this is not adding support for mac, but removing support for x64.

File: docker-compose.yml [hunk 1]
```diff
@@ -186,7 +186,7 @@ services:
       test: ["CMD-SHELL", "nc -z localhost 9092"]
   clickhouse:
     <<: *restart_policy
-    image: "yandex/clickhouse-server:20.3.9.70"
+    image: "altinity/clickhouse-server:21.12.3.32.altinitydev.arm"
```
[the remark this answers is not in the document]
Comment (waterfallmouse) [reply docker-compose.yml:189]: @aminvakil 
I'm sorry, but I don't know because I only checked this branch with m1 mac.

**Comment by aminvakil** [comment]
Thanks for this, but we clearly cannot change this repository and break every other user not using an arm CPU :)

Please make a PR working for both x64 and arm architectures and we can discuss on it.

**Pull request closed without merging**